common/schemas.ts

```diff
@@ -10,6 +10,7 @@ enum invitation_status {
   COMPLETE = "COMPLETE",
 }
 
+
 export const GithubInfoSchema = z.object({
```

Comment:
Was there a need to include the extra space in the commit? As a reminder - would always review the changes in a PR after submitting it, and update if needed

With syntax you are doing the right thing here, types, interfaces, classes, etc. should always be capitalized camel case (ie CapitalizedCamelCase)